src/components/SelectBox.vue

```diff
@@ -33,6 +33,7 @@ export default {
 @import "src/styles/variables";
 
 .select-box {
+  $box: &;
```

Comment:
wat doet dit?

Reply:
zo kun je het hoofd element gebruiken in een variabel nu is $box het zelfde als .select-box, Maar volgens mij gebruik ik het niet meer in de class dus kan weer weg.